examples/gallery/image_bright_regions_gallery_example.py

```diff
@@ -6,15 +6,17 @@
 Manipulating Map image data - finding bright regions
 ====================================================
 
-This example shows how you can do basic image processing on SunPy map image data. In this example, we try to find the brightest regions in an AIA image, and count the approximate number of regions of interest.
+This example shows how you can do basic image processing on SunPy map image data.
+In this example, we try to find the brightest regions in an AIA image, and count the approximate number of regions of interest.
 """
 
 ##############################################################################
 # First, import the modules we will need:
 
 
 import sunpy
-from sunpy import map
+from sunpy import map as smap
```

Comment:
please import map like `import sunpy.map` for consistency with the rest of the docs.